tests/unit/composables/Project.spec.js

```diff
@@ -250,4 +253,18 @@ describe('Test composable: Project', () => {
       ]);
     });
   });
+
+  describe('Test function: newBranchFrom', () => {
+    it('Should not call checkout without checkout demand', async () => {
+      const result = await newBranchFrom('test', 'branch', 'main', false);
+      expect(result).toEqual('CheckoutEventNext');
+      expect(git.checkout).not.toBeCalled();
+    });
+
+    it('Should call checkout with checkout demand', async () => {
+      const result = await newBranchFrom('test', 'branch', 'main', true);
+      expect(result).toEqual('CheckoutEventNext');
```

Comment:
`checkout: jest.fn(() => Promise.resolve('checkout'))`
Agree, should return `checkout`

Reply:
because, in one case we call checkout from Project that also call the checkout event.
I think line 267 is what you want to have. So I think I don't have to do anything.

Reply:
ok for me

Reply:
ok